### PR TITLE
(VANAGON-121) Prep for 0.15.5 release

### PR DESCRIPTION
**Pull request by mwaggett**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+##[0.15.5] - released on 2018-03-05
+### Changed
```
Comment (underscorgan) [inline CHANGELOG.md]: This should probably be `Fixed`, while it's a change it's really a bugfix more than a change in behavior.

Comment (mwaggett) [reply CHANGELOG.md]: fair enough

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+##[0.15.5] - released on 2018-03-05
+### Fixed
```
Comment (mwaggett) [inline CHANGELOG.md]: @underscorgan Fixed (see what I did there 😜)

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,11 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+##[0.15.5] - released on 2018-03-05
```
Comment (sarameisburger) [inline CHANGELOG.md]: formatting looks wonky, i think you need a space between the ## and [

Comment (mwaggett) [reply CHANGELOG.md]: whoops!

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,11 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.15.5] - released on 2018-03-05
```
Comment (mwaggett) [inline CHANGELOG.md]: @sarameisburger updated!

**Pull request merged**